css/amp.css

```diff
@@ -287,6 +287,17 @@ template {
   display: none !important;
 }
 
+/**
+ * Provide common border-box sizing pattern to authors
+ */
+html.amp-border-box {
+  box-sizing: border-box;
+}
+html.amp-border-box *,
+html.amp-border-box *:before,
+html.amp-border-box *:after {
+  box-sizing: inherit;
```

Comment:
Hmm. By default though `box-sizing` is not inherited. This could cause some unexpected behavior where someone would override box-sizing just for one element and it'd automatically apply to everything within.

What about if we just explicitly state `border-box` instead of `inherit`?